docs/administration/upgrade/to_v6.0.adoc

```diff
@@ -337,71 +373,212 @@ apiVersion: "observability.openshift.io/v1"
-    type: observability.openshift.io/Authorized
-  - message: ""
-    reason: Validation Success
-    status: "True"
-    type: observability.openshift.io/Valid
-  inputs:
-  - message: ""
-    status: "True"
-    type: observability.openshift.io/Valid-application
-  outputs:
-  - message: ""
-    status: "True"
-    type: observability.openshift.io/Valid-rh-loki
-  pipelines:
-  - message: ""
-    status: "True"
-    type: observability.openshift.io/Valid-application-logs
+    - message: "permitted to collect log types: [application]"
+      reason: ClusterRoleExists
+      status: "True"
+      type: observability.openshift.io/Authorized
+    - message: ""
+      reason: ValidationSuccess
+      status: "True"
+      type: observability.openshift.io/Valid
+    - message: ""
+      status: "True"
+      type: observability.openshift.io/Ready
+  filtersStatus:
+    - message: filter "my-parse" is valid
+      reason: ValidationSuccess
+      status: "True"
+      type: observability.openshift.io/Valid-my-parse
+  inputsStatus:
+    - message: input "application" is valid
+      reason: ValidationSuccess
+      status: "True"
+      type: observability.openshift.io/Valid-application
+  outputsStatus:
+    - message: output "rh-loki" is valid
+      reason: ValidationSuccess
+      status: "True"
+      type: observability.openshift.io/Valid-rh-loki
+  pipelinesStatus:
+    - message: pipeline "app-logs" is valid
+      reason: ValidationSuccess
+      status: "True"
+      type: observability.openshift.io/Valid-app-logs
 ----
 
-NOTE: Conditions that are satisfied and which apply have a "status" value of "True".  Conditions that
-have a "status" other than "True" provide a reason and a message identifying why.
-
-=== Examples & Common Use Cases
+NOTE: Conditions that are satisfied have a "status" value of "True".  Conditions that
+have a "status" other than "True" provide a reason and a message identifying the failure.
 
-==== ClusterLogging Only
-===== Red Hat Managed Elasticsearch
-===== Red Hat Managed LokiStack
+.6.0 Status "False" Example
+[source, yaml]
+----
+status:
+  conditions:
+    - message: insufficient permissions on service account, not authorized to collect 'application' logs
+      reason: ClusterRoleMissing
+      status: "False"
+      type: observability.openshift.io/Authorized
+  inputsStatus:
+    - message: input "application" is valid
+      reason: ValidationFailure
+      status: "False"
+      type: observability.openshift.io/Valid-application
+----
+== Examples & Common Use Cases
 
-==== ClusterLogging & ClusterLogForwarder
-===== Red Hat Managed Elasticsearch
-===== Red Hat Managed LokiStack
-===== Forwarding to CloudWatch using Short-Lived Token
-===== Forwarding to Elasticsearch using Custom Indices
-===== Forwarding with Input Receivers
+=== ClusterLogForwarder Only
+====
+.Forwarding to CloudWatch using Short-Lived Token (SA Token)
+[source, yaml]
+----
+apiVersion: observability.openshift.io/v1
```

Comment:
This is a place for us to promote reuse and possible add tests to ensure they are valid.  Recommend moving this to separate files and using an include (i.e.) something like:
```
 .v6.0 Forwarding to Red Hat Managed LokiStack
[source,yaml]
 ----
include::../../../docs/reference/samples/forward-to-rh-managed-es.yaml[]
 ----
```